mgmtfn/dockplugin/netDriver.go

```diff
@@ -762,14 +774,6 @@ func deleteNetworkHelper(networkID string) error {
 	} else {
 		log.Infof("Could not find contiv network %v", networkID)
 	}
-	dnet, err := docknet.FindDocknetByUUID(networkID)
-	if err == nil {
-		// delete the dnet oper state
-		err = docknet.DeleteDockNetState(dnet.TenantName, dnet.NetworkName, dnet.ServiceName)
-		if err != nil {
-			log.Errorf("Couldn't delete docknet for nwID %s", networkID)
-		}
-		log.Infof("Deleted docker network mapping for %v", networkID)
-	}
+
```

Comment:
we deleted docker network, found a contiv netowrk, but failed to delete the contiv netowrk, should we restore the docker network? (i dunno, just asking) . .  L771

Reply:
There is no hook for me to do that at the moment to restore the network.

Reply:
Don't think we must secretly revert what the user does (`docker network rm` in this case).  If contiv network deletion fails, I think an error to the user is good instead of reverting what the user did.

Reply:
OK